### PR TITLE
Move search to core/search

### LINKED ISSUE #13259: Search results are not localized
I am unable to find it why, but search.js strings are not parsed for translation. They are not on transifex either.

https://github.com/owncloud/core/blob/master/search/js/search.js#L211

### PR DESCRIPTION
**Pull request by MorrisJobke**
fixes #13259

cc @LukasReschke @butonic @DeepDiver1975

### CODE REVIEW

**Comment by MorrisJobke** [comment]
@DeepDiver1975 Does it now scan the files for l10n or is something else needed?

File: core/routes.php [hunk 1]
```diff
@@ -22,8 +22,8 @@
 /** @var $this OCP\Route\IRouter */
 // Core ajax actions
 // Search
-$this->create('search_ajax_search', '/search/ajax/search.php')
-	->actionInclude('search/ajax/search.php');
+$this->create('search_ajax_search', '/core/search/ajax/search.php')
```
Comment (LukasReschke) [inline core/routes.php]: When we already touch this it would be soooo awesome if we could get rid of the .php at the end ;-)

Comment (MorrisJobke) [inline core/routes.php]: Done :)

**Comment by MorrisJobke** [comment]
@owncloud-bot retest this please

**Comment by MorrisJobke** [comment]
JS error -.- I'm on it

**Comment by DeepDiver1975** [comment]
> @DeepDiver1975 Does it now scan the files for l10n or is something else needed?

no - core is scanned for translation strings - thx

**Comment by DeepDiver1975** [comment]
@MorrisJobke 

````
00:38:53.228 [32mINFO [karma]: [39mKarma v0.12.31 server started at http://localhost:9876/
00:38:53.232 [32mINFO [launcher]: [39mStarting browser PhantomJS
00:38:53.240 [33mWARN [watcher]: [39mPattern "/var/jenkins/workspace/pull-request-analyser-ng-simple/label/SLAVE/search/js/search.js" does not match any file.
00:38:54.105 [32mINFO [PhantomJS 1.9.8 (Linux)]: [39mConnected on socket lWUtHm2w1vRGiNnwMOcN with id 21104545
00:38:54.444 [31mPhantomJS 1.9.8 (Linux) ERROR[39m
00:38:54.444   TypeError: 'undefined' is not an object (evaluating 'OCA.Search.Files = Files')
00:38:54.444   at /var/jenkins/workspace/pull-request-analyser-ng-simple/label/SLAVE/apps/files/js/search.js:189
00:38:54.448 PhantomJS 1.9.8 (Linux): Executed 0 of 0[31m ERROR[39m (0.342 secs / 0 secs)
00:38:54.699 Build step 'Conditional step (single)' marked build as failure
00:38:54.827 Recording test results

**Comment by DeepDiver1975** [comment]
^^^ JS still needs some love

**Comment by MorrisJobke** [comment]
Thanks for the reminder. Forgot about that one

**Comment by MorrisJobke** [comment]
> > Does it now scan the files for l10n or is something else needed?
> 
> no - core is scanned for translation strings - thx


This was an "or" question - you can't answer with yes or no ;)

**Comment by MorrisJobke** [comment]
@DeepDiver1975 Fixed

**Comment by MorrisJobke** [comment]
@owncloud-bot retest this please

**Comment by DeepDiver1975** [comment]
tested - search still works :+1: 

@butonic might want to have a look as well - THX

**Comment by DeepDiver1975** [comment]
@butonic are the files result.css and the partial template still needed? Please have a look - THX

**Comment by MorrisJobke** [comment]
> @butonic are the files result.css and the partial template still needed? Please have a look - THX

Yes. They are needed. I also tried to get rid of them ;)

**Comment by LukasReschke** [comment]
:+1:

**Pull request merged**